src/es/components/molecules/teaser/square-image-/square-image-.css

```diff
@@ -1,11 +1,11 @@
 :host {
+    display: block;
     position: relative;
     color: white;
     --color: currentColor;
 
     font: var(--mdx-sys-font-fix-label0);
     min-width: 200px;
-    max-width: 230px;
```

Comment:
A variable might be appropriate here to be able to control the values ​​via variablesCustom.css. What do you think?